pyphare/pyphare_tests/test_pharesee/test_geometry.py

```diff
@@ -355,6 +425,200 @@ def test_level_ghost_boxes_do_not_overlap_patch_interiors(self):
 
 
 
+@ddt
+class SimpleGeometryTest(unittest.TestCase):
```

Comment:
not sure why there is a kind of duplication of the tests here.
Maybe there should be only one fixture.
I like the fact that all quantities are tested though.

Reply:
resolution determined on slack - Merge test classes with SimpleGeometryTest type test cases

Reply:
done